### ISSUE TITLE
Record is not support

### ISSUE THREAD

**Issue by bluescurry** [d-kimuson/ts-type-expand#372]
declare type IUser = Record<'name' | 'phone', string>;
![image](https://github.com/d-kimuson/ts-type-expand/assets/19237129/9f8bc322-479b-4950-a825-2d979e1c2f3b)

**Comment by TonyGravagno** [comment]
I'm seeing this too. To help in reproducing this, here is my example:

```ts
// Define a union of string literals for the keys
type StyleKeys =
  | 'labelledControl'
  | 'submitButton'
  | 'formContainer'
  | 'formFieldRow'
// more...
// Use the Record utility type to create a type with those keys, all having the same value type
type ReactStyles = Record<StyleKeys, CSS.Properties>
...
react: ReactStyles
...
```
![image](https://github.com/d-kimuson/ts-type-expand/assets/499473/dcb6b163-73f6-4ddd-8531-61d217896cee)

Another question arises here - for a Record<> type, do we really want this extension to build the entire list of details for every property/record? That is:
![image](https://github.com/d-kimuson/ts-type-expand/assets/499473/076a909f-b3bb-4eff-99e6-a81217965d8a)

I think not. If the entire node is composed entirely of properties that are all of the same complex type, I'm thinking it would be fine to have a single subnode off of the root, maybe named "_" that has all of the properties of type T ( CSS.Properties above ) and then the list of keys can just be shown as Keys (K extends string | number | symbol) without themselves being parent nodes with children (like for a simple union of literals or Boolean values) : 
![image](https://github.com/d-kimuson/ts-type-expand/assets/499473/ee08ff49-1ac8-49e0-9322-6a597f3e31cd)

HTH

**Comment by d-kimuson** [comment]
@bluescurry, @TonyGravagno 
Thank you for your reports!
I have included this fix in the new release ([v1.1.0](https://github.com/d-kimuson/ts-type-expand/releases/tag/v1.1.0)), so please check it out!

**Comment by d-kimuson** [comment]
Close.
